framework/src/wifi_manager/Kconfig

```diff
@@ -70,5 +70,22 @@ config WIFIMGR_DISABLE_AUTO_GET_IP
 	bool "disable auto get ip (ipv4 dhcp)"
 	default n
 
+config WIFIMGR_DISABLE_EXTERNAL_AUTOCONNECT
+	bool "disable external auto(re)connect of wifi driver"
+	default n
+
+if WIFIMGR_DISABLE_EXTERNAL_AUTOCONNECT
+config WIFIMGR_ENABLE_INTERNAL_AUTOCONNECT
```

Comment:
Do we need this 'WIFIMGR_DISABLE_EXTERNAL_AUTOCONNECT'. I think WIFIDRIVER_SUPPORT_AUTOCONNECT can do same thing.

Reply:
@pillip8282 That is right. However, WIFIDRIVER_SUPPORT_AUTOCONNECT is defined in wifi_manager.c which is an internal variable and should be mapped to CONFIG_xxx, if it is controlled by the menuconfig.